keps/sig-storage/661-statefulset-volume-resize/kep.yaml

```diff
@@ -0,0 +1,51 @@
+title: StatefulSet volume resize
+kep-number: 661
+authors:
+  - "@areller"
+  - "@harshanarayana"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-storage
```

Comment:
Based on the current location of the proposal, you should make sig-storage as the owning-sig and sig-apps as participating.